packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -75,12 +80,15 @@ export class WorkspaceQuickOpenItem extends QuickOpenItem {
     }
 
     /**
-     * Display the workspace path as part of
-     * the WorkspaceQuickOpenItem description
+     * Display shortened workspace path based on user's operating system
      * @returns workspace path
      */
     getDescription(): string {
-        return (this.workspace) ? new URI(this.workspace).path.dir.toString() : '';
+        let path = (this.workspace) ? new URI(this.workspace).path.dir.toString() : '';
+        if (this.platform !== 'win32' && path.includes(this.home)) {
```

Comment:
`includes`?  What if my home is `/home/simark` and the workspace is `/backups/pc/home/simark/foo`?

Reply:
Another corner-case test to add: path `/home/simark123/foo` and home `/home/simark`.

Reply:
nice catch! I should only replace the result of the user's `home` if it is at the start of the path.